src/java.base/share/classes/java/text/package-info.java

```diff
@@ -26,7 +26,7 @@
 /**
  * Provides classes and interfaces for handling text, dates, numbers,
  * and messages in a manner independent of natural languages.  This
- * means your main application or applet can be written to be
+ * means your application can be written to be
```

Comment:
Maybe we should be broadened a bit, e.g  "an application or library" instead of "your application".

Reply:
Thanks. Broadened.